packages/profile-sync-controller/src/sdk/authentication.ts

```diff
@@ -50,6 +58,29 @@ export class JwtBearerAuth implements SIWEInterface, SRPInterface {
     return await this.#sdk.signMessage(message);
   }
 
+  async pairIdentifiers(
+    pairing: Pair[],
+    signMessage: (message: string) => Promise<string>,
+  ): Promise<void> {
+    const profile = await this.getUserProfile();
+    const n = await getNonce(profile.profileId, this.#env);
+    const logins = await Promise.all(
```

Comment:
This is fine. Was thinking of `Promise.allSettled()`, but I think we want to error if 1 of the logins fail.